.gitignore

```diff
@@ -0,0 +1 @@
+/doc/tags
```

Comment:
I'd like to be explicit here, leading slash prevents git from being overly aggressive and ignoring files of that name at every level of the repo.